modules/ding_availability/js/ding_availability.js

```diff
@@ -114,6 +114,7 @@
         // Insert/update holding information for material.
         var holdings = Drupal.DADB[entity_id];
         $('#' + id).html(holdings.html);
+        Drupal.attachBehaviors(holdings.html);
```

Comment:
Is it correct to call `Drupal.attachBehaviors()` on the returned HTML and not on the `$('#' + id)` HTML element to update?

Reply:
Yes, passing `$('#' + id)` would be the correct here. Updated PR!